enigmatry-entry-blueprint-app/jest.config.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+    "roots": [
+        "<rootDir>"
+    ],
+    "modulePaths": [
+      "<rootDir>"
+    ],
+    "preset": 'jest-preset-angular',
+    "setupFilesAfterEnv": [
+      '<rootDir>/src/testing/setup.ts',
+      '<rootDir>/src/testing/mocks/console.ts'
+    ],
+    "globalSetup": 'jest-preset-angular/global-setup',
+    "collectCoverage": true,
+    "collectCoverageFrom": [
+        "<rootDir>/src/app/core/**/*.ts",
```

Comment:
Angular just added experimental support for Jest in v16. I don't blame them we're not having strong recommendations from their side. On the other hand, one of advantages Jest has over Karma is that it doesn't need full browser to run since it's node-based. That'd mean tests are executed much faster since Jest is optimized for unit testing.

Even though everything's possible, for testing DOM interactions (components) Jest isn't the best tool. No native support for angular environment + you can observe Interceptor test to have an idea how many things are needed just to run simple, inefficient and slow test. Luckily there are other frameworks that are more suitable and optimized for DOM testing such as Playwright or Cypress and we can use those to write integration/e2e FE tests.